test/filling-method.js

```diff
@@ -81,6 +87,7 @@ describe('filling method', function() {
           'topY': 0,
           'width': 615,
           'height': 370,
+          'toBeBlurred': true,
```

Comment:
There are no tests with 'toBeBlurred: false'. Could you also add a test that simulates the click to check that your toggle function works correctly?